addons/web/static/src/core/browser/feature_detection.js

```diff
@@ -48,6 +48,10 @@ export function isAndroidApp() {
     return /OdooMobile.+Android/i.test(browser.navigator.userAgent);
 }
 
+export function isPWA() {
```

Comment:
Would just change this name to not include "PWA" (as it's a compound of different technologies).
```suggestion
export function isDisplayStandalone() {
```